CONTRIBUTING.adoc

```diff
@@ -18,7 +18,7 @@ Thank you for your contributions! Please follow this process to submit a patch:
 . Fork the repository and create a topic branch.
 . Refer to the link:https://docs.openshift.org/latest/minishift/contributing/developing.html[developer documentation] for guidelines and tips on how to build and test Minishift.
 . Submit a pull request with the proposed changes.
-  To make reviewing and integration as smooth as possible, please run the `make` targets link:https://docs.openshift.org/latest/minishift/contributing/developing.html#unit-tests[`test`], https://docs.openshift.org/latest/minishift/contributing/developing.html#format-source-code[`fmtcheck`] and https://docs.openshift.org/latest/minishift/contributing/developing.html#commit-messages[`validate_commits`] prior to submitting the pull request.
+  Format for Commit message: *Issue #<issue_no> message*. The regex `^(Issue #[0-9]+ .*|cut v[0-9]+\.[0-9]+\.[0-9]+)$$` is used to validate commit messages. To make reviewing and integration as smooth as possible, please run the `make` targets link:https://docs.openshift.org/latest/minishift/contributing/developing.html#unit-tests[`test`], https://docs.openshift.org/latest/minishift/contributing/developing.html#format-source-code[`fmtcheck`] and https://docs.openshift.org/latest/minishift/contributing/developing.html#commit-messages[`validate_commits`] prior to submitting the pull request.
```

Comment:
Is it OK to put the regex here? its for contributors, they would know exactly what pattern they have to match.

Reply:
Agree. Regex is not required here

Reply:
@anjannath In the current changes are not acceptable as makes the bullet point almost like a paragraph. Please break it in to two bullet points under `Submit a pull request with the proposed changes under `Submit a pull request with the proposed changes`. 
e.g. 
* Submit a pull request with the proposed changes
  * The required format for the git commit message should be *Issue #<issue_no> commit message*
  *  To make reviewing and integration as smooth as possible, please run the make targets test, fmtcheck and validate_commits prior to submitting the pull request.

Reply:
Okay @LalatenduMohanty